plugins/bitwarden-software-engineer/README.md

```diff
@@ -0,0 +1,66 @@
+# Bitwarden Software Engineer Plugin
+
+A comprehensive full-stack development plugin for Claude Code with language-specific skills and specialized agents.
+
+## What This Plugin Provides
+
+**Skills** that disclose to Claude language patterns and conventions:
+- TypeScript, C#, Rust, SQL development patterns
+- Progressive loading: only relevant skills activate when needed
+
+**Agents** that coordinate complex workflows across languages:
+- Architecture design, security review, code review, performance optimization
```

Comment:
"Architecture" feels out of scope for this, and is a separate agent. The engineer must lean on already-defined architectures.